.gitignore

```diff
@@ -226,3 +226,5 @@ Results
 Package
 .DS_Store
 .idea
+/src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Logger/bin
```

Comment:
That is where the output of the Backup projects build to. Unfortunately, they aren't being covered by some other ignore statement currently, so during testing, I'd have dozens of uncommitted files. So, I'm ignoring them since they shouldn't ever be checked in.